crates/apollo-schema-index/src/traverse.rs

```diff
@@ -53,67 +53,65 @@ impl SchemaExt for Schema {
+                                    let field_type = field.ty.inner_named_type();
+                                    let field_args = field
+                                        .arguments
+                                        .iter()
+                                        .map(|arg| arg.ty.inner_named_type().clone())
+                                        .collect::<Vec<_>>();
+                                    (
+                                        field_type,
+                                        current_path.clone().add_child(
+                                            Some(field.name.clone()),
+                                            field_args,
+                                            field_type.clone(),
+                                        ),
+                                    )
+                                }));
+                            }
+                            ExtendedType::Union(union) => {
+                                stack.extend(union.members.iter().map(|member| &member.name).map(
+                                    |next_type| {
                                         (
-                                            field_type,
+                                            next_type,
                                             current_path.clone().add_child(
-                                                Some(field.name.clone()),
-                                                field_args,
-                                                field_type.clone(),
+                                                None,
+                                                vec![],
+                                                next_type.clone(),
                                             ),
                                         )
-                                    }));
-                                }
-                                ExtendedType::Union(union) => {
-                                    stack.extend(
-                                        union.members.iter().map(|member| &member.name).map(
-                                            |next_type| {
-                                                (
-                                                    next_type,
-                                                    current_path.clone().add_child(
-                                                        None,
-                                                        vec![],
-                                                        next_type.clone(),
-                                                    ),
-                                                )
-                                            },
-                                        ),
-                                    );
-                                }
-                                _ => {}
+                                    },
+                                ));
```

Comment:
I'm not sure I see how this change relates to the rust-toolchain or Renovate updates. Is this addressing something else?

Reply:
Not using let chains now fails clippy, so I had to update a few of those for this PR.